docs/getting-started/migrating-to-v7.md

```diff
@@ -4,10 +4,12 @@ In version 7.x we got rid of some old habits, leading to several breaking change
 
 ## No automatic query of all node values when restarting from cache
 
-This legacy behavior resulted in a lot of traffic and delays when the driver was restarted, often doing many unnecessary queries. To reduce the strain on battery devices and to keep the network as responsive as possible, we've opted not to do this anymore when a node was previously interviewed. This also means that the `"interview completed"` event is no longer emitted on restart.
+This legacy behavior resulted in a lot of traffic and sdelays when the driver was restarted, often doing many unnecessary queries. To reduce the strain on battery devices and to keep the network as responsive as possible, we've opted not to do this anymore when a node was previously interviewed. This also means that the `"interview completed"` event is only emitted after the initial interview and after manually-requested re-interviews, but not after each restart.
```

Comment:
> This legacy behavior resulted in a lot of traffic and sdelays

s in delays

Reply:
that's a typo. Probably wanted to save the document :D